Own Animation/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>
+            Own Animation
+        </title>
+        <link rel="stylesheet" href="/Own Animation/styles.css">
+    </head>
+    <body>
+
+        <!-- START : BIG WHITE CIRCLE -->
+        <div class="bigCircle">
```

Comment:
Use kebab-case for all the clasess in the app

Reply:
converted all class names to kebab case ...